### ISSUE TITLE
TkAppITCase.rendersHtmlByDefault fails locally

### ISSUE THREAD

**Issue by komaz** [yegor256/netbout#635]
The test fails with a message:
```
Failed tests:
  TkAppITCase.rendersHtmlByDefault:113 HTTP response status is not equal to 200
```

An exception happens on server side:
```
java.io.IOException: net.sf.saxon.trans.XPathException: java.net.MalformedURLException: unknown protocol: about
	at org.takes.rs.RsPrettyXML.transform(RsPrettyXML.java:119)
	at org.takes.rs.RsPrettyXML.make(RsPrettyXML.java:89)
	at org.takes.rs.RsPrettyXML.head(RsPrettyXML.java:73)
	at org.takes.rs.RsWrap.head(RsWrap.java:58)
	at org.takes.rs.RsWithHeader$1.head(RsWithHeader.java:107)
	at org.takes.rs.RsWrap.head(RsWrap.java:58)
	at org.takes.rs.RsWrap.head(RsWrap.java:58)
	at org.takes.facets.forward.TkForward$Safe.load(TkForward.java:112)
	at org.takes.facets.forward.TkForward$Safe.body(TkForward.java:99)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:132)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:111)
	at org.takes.rs.RsWrap.body(RsWrap.java:63)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:111)
	at org.takes.rs.RsWrap.body(RsWrap.java:63)
	at org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:68)
	at org.takes.rs.RsWrap.body(RsWrap.java:63)
	at org.takes.rs.RsWrap.body(RsWrap.java:63)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:158)
	at org.takes.rs.RsPrint.print(RsPrint.java:111)
	at org.takes.http.BkBasic.print(BkBasic.java:99)
	at org.takes.http.BkBasic.accept(BkBasic.java:77)
	at org.takes.http.BkSafe.accept(BkSafe.java:58)
	at org.takes.http.BkTimeable.accept(BkTimeable.java:94)
	at org.takes.http.BkParallel$1.run(BkParallel.java:97)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: net.sf.saxon.trans.XPathException: java.net.MalformedURLException: unknown protocol: about
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:424)
	at net.sf.saxon.event.Sender.send(Sender.java:214)
	at net.sf.saxon.IdentityTransformer.transform(IdentityTransformer.java:30)
	at org.takes.rs.RsPrettyXML.transform(RsPrettyXML.java:117)
	... 26 more
Caused by: java.net.MalformedURLException: unknown protocol: about
	at java.net.URL.<init>(URL.java:592)
	at java.net.URL.<init>(URL.java:482)
	at java.net.URL.<init>(URL.java:431)
	at com.sun.org.apache.xerces.internal.impl.XMLEntityManager.setupCurrentEntity(XMLEntityManager.java:616)
	at com.sun.org.apache.xerces.internal.impl.XMLEntityManager.startEntity(XMLEntityManager.java:1297)
	at com.sun.org.apache.xerces.internal.impl.XMLEntityManager.startDTDEntity(XMLEntityManager.java:1264)
	at com.sun.org.apache.xerces.internal.impl.XMLDTDScannerImpl.setInputSource(XMLDTDScannerImpl.java:263)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$DTDDriver.dispatch(XMLDocumentScannerImpl.java:1164)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$DTDDriver.next(XMLDocumentScannerImpl.java:1050)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next(XMLDocumentScannerImpl.java:964)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:117)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:510)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:648)
	at net.sf.saxon.event.Sender.sendSAXSource(Sender.java:404)
	... 29 more
```
Looks like `RsPrettyXML` can't format produced HTML, more specifically its `SYSTEM` property in `DOCTYPE`:
```xml
<!DOCTYPE html
  SYSTEM "about:legacy-compat">
<html>
...
</html>
```

This issue is also reproducible by running netbout with `mvn clean install -Phit-refresh -Dport=8080` and opening a local server with Firefox.

**Comment by dmzaytsev** [comment]
@komaz fixed in #636

**Comment by komaz** [comment]
@dmzaytsev great
